shanoir-ng-studies/src/main/java/org/shanoir/ng/messaging/StudyUserUpdateBroadcastService.java

```diff
@@ -34,14 +35,11 @@
 	@Autowired
 	private RabbitTemplate rabbitTemplate;
 	
-	
 	public void broadcast(Iterable<StudyUserCommand> commands) throws MicroServiceCommunicationException {
 		try {
 			String str = new ObjectMapper().writeValueAsString(commands);
-			rabbitTemplate.convertAndSend("study-user-exchange", "study-user", str);
-			
+			rabbitTemplate.convertAndSend(RabbitMQConfiguration.STUDY_USER_EXCHANGE, "study-user", str);
```

Comment:
Is  "study-user" a queue ? Maybe we can define this constant in RabbitMQConfiguration too ?

Reply:
STUDY_USER_QUEUE

Reply:
done, part of my next commit